pkg/grpc/grpc_api.go

```diff
@@ -54,6 +54,7 @@ type ApplicationContext struct {
 	PubSubs               map[string]pubsub.PubSub
 	StateStores           map[string]state.Store
 	Files                 map[string]file.File
+	Oss                   map[string]file.Oss
```

Comment:
I prefer to add a separate API plugin for s3 API instead of puting it in the default API plugin